docs/area-owners.md

```diff
@@ -1,122 +1,120 @@
+| area-Microsoft.VisualBasic                     | @jaredpar     | @cston @333fred                                     | Archived component - limited churn/contributions (see [#33170](https://github.com/dotnet/corefx/issues/33170))                                                                                                                                                                            |
+| area-Microsoft.Win32                           | @ericstj      | @maryamariyan @Anipik                               | Including System.Windows.Extensions                                                                                                                                                                                                                                                       |
+| area-PAL-coreclr                               | @mangod9      | @janvorli                                           |                                                                                                                                                                                                                                                                                           |
+| area-Performance-mono                          | @SamMonoRT    | @SamMonoRT                                          |                                                                                                                                                                                                                                                                                           |
+| area-R2RDump-coreclr                           | @mangod9      | @nattress                                           |                                                                                                                                                                                                                                                                                           |
+| area-ReadyToRun-coreclr                        | @mangod9      | @nattress                                           |                                                                                                                                                                                                                                                                                           |
+| area-SDK                                       | @mangod9      | @janvorli                                           | General development issues and overlap with the SDK and CLI                                                                                                                                                                                                                               |
```

Comment:
This is more of leecoward

Reply:
Thanks, I should also note that janvorli is not the right sub owner either.  Lee hopefully would know who is.

Reply:
Well, I put @leecow for now.